docs/sql-ref-syntax-aux-show-columns.md

```diff
@@ -18,5 +18,26 @@ license: |
   See the License for the specific language governing permissions and
   limitations under the License.
 ---
+### Description
+Return the list of columns in a table. If the table does not exist, an exception is thrown.
 
-**This page is under construction**
+### Syntax
+{% highlight sql %}
+SHOW COLUMNS {IN | FROM} [db.]table {IN | FROM} database
+{% endhighlight %}
+**Note**
+- You can list the columns of a table in a database other than current database in one of following
```

Comment:
I think this should be `You can list the columns of a table in a database other than current database in one of the following ways:`